ipaserver/plugins/baseuser.py

```diff
@@ -123,6 +123,24 @@ def fix_addressbook_permission_bindrule(name, template, is_new,
         template['ipapermbindruletype'] = 'anonymous'
 
 
+def update_samba_attrs(ldap, dn, entry_attrs, options):
+    samba_attrs = {'ipantlogonscript', 'ipantprofilepath',
+                   'ipanthomedirectory', 'ipanthomedirectorydrive'}
+
+    for attr in samba_attrs:
+        if options.get(attr, None):
+            if attr in entry_attrs['objectclass']:
```

Comment:
This is wrong -- attribute value cannot be in the `objectclass`. I suspect you wanted to check that `ipaNTUserAttrs` is in the `objectclass`.